src/api/tests/test_profile.py

```
@@ -16,3 +18,51 @@ def test_add_profile_item_quick_creates_definition(app):
 
         definitions = get_profile_item_definition_list(app, "course-1")
         assert any(item.profile_key == "favorite_color" for item in definitions)
+
+
+def test_hide_unused_profile_items_no_unused(monkeypatch):
+    calls = []
+
+    def fake_get_unused(app, parent_id):
```

Ruff (0.14.13)

Unused function argument: `app`

(ARG001)